.changeset/funny-rabbits-think.md

```diff
@@ -0,0 +1,6 @@
+---
+"@swisspost/design-system-documentation": minor
+"@swisspost/design-system-styles": minor
+---
+
+The CSS `.form-check-sm` class has been added to the Form Radio Button component and the variant size is added to the component's documunetation.
```

Comment:
Can you rephrase the message? Changeset entries must start with a verb in past tense, like "Added [...]."